dev-docs/modules/dfp_video.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Module - Dfp Video
+description: Addition of DFP Video to the Prebid package
+top_nav_section: dev_docs
+nav_section: modules
+module_code : dfpAdServerVideo
+display_name : Dfp Video
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# DFP Video Module
+{:.no_toc}
+
+This module is required to use the Prebid Instream video examples with DFP Adserver.
+
+## Instructions for adding Dfp Video code to a Prebid package
```

Comment:
I think this header text can be removed and a sentence added to the previous paragraph something like:

<blockquote>
This module is required to use the Prebid Instream video examples with DFP Adserver.  For instructions showing how to add this module to Prebid.js, see below.
</blockquote>